result/convergence.yaml

```diff
@@ -0,0 +1,102 @@
+components:
+  schemas:
+    Convergence.Request:
+      description: >-
+        The container for a CPDP convergence request.
+      type: object
+      properties:
+        flow_names:
+          description: |-
+            Flow metrics will be retrieved for these flow names.
+            If no flow names are specified then all flows will be returned.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/Flow/properties/name"
+          x-field-uid: 1
+    Convergence.Metric:
+      description: |-
+        The container for convergence measurements
+      type: object
+      properties:
+        name:
+          description: >-
+            The name of a flow. 
+          type: string
+          x-field-uid: 1
+        data_plane_convergence_us:
+          description: |-
+            The convergence time measured from the data plane perspective only. 
+            It measures the time from BTT, when the rate on Test Port 2 crosses 
```

Comment:
The time is relative to what ... last start of traffic of the affected flow ? Good to also mention time ( in microsecs ? )

Reply:
Need discussion.